doc/builders/fetchers.chapter.md

```diff
@@ -47,24 +47,131 @@ A number of fetcher functions wrap part of `fetchurl` and `fetchzip`. They are m
 
 ## `fetchFromGitHub`
 
-`fetchFromGitHub` expects four arguments. `owner` is a string corresponding to the GitHub user or organization that controls this repository. `repo` corresponds to the name of the software repository. These are located at the top of every GitHub HTML page as `owner`/`repo`. `rev` corresponds to the Git commit hash or tag (e.g `v1.0`) that will be downloaded from Git. Finally, `sha256` corresponds to the hash of the extracted directory. Again, other hash algorithms are also available but `sha256` is currently preferred.
+Fetch a Git repository from [GitHub](https://github.com/).
+
+### Required arguments
+
+`owner`
+
+: GitHub user or organization that controls this repository.
```

Comment:
The colon as a bullet point looks a little weird to me (in the rendered markdown). Maybe use the typical `*`, or nothing at all?

Reply:
This is [pandoc syntax](https://pandoc.org/MANUAL.html#definition-lists), it will be rendered correctly in the manual.

Reply:
Oh, I see. Thanks for the correction.